src/entries/background/index.ts

```diff
@@ -18,6 +20,26 @@ import { handleTabAndWindowUpdates } from './handlers/handleTabAndWindowUpdates'
 import { handleWallets } from './handlers/handleWallets';
 require('../../core/utils/lockdown');
 
+declare const browser: Browser;
+
+const browserType = process.env.BROWSER;
```

Comment:
This might not work in a prod build; I think we only use this for e2e/local builds

Reply:
We have a `getBrowser` util here: https://github.com/rainbow-me/browser-extension/blob/97b914a299f1454b30b6ffbd87080929a2c1f097/src/entries/popup/hooks/useBrowser.ts#L8C17-L8C27